docs/pages/what-is-teleport.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: What is Teleport?
+description: Provides an overview of the purpose and benefits of Teleport Access Platform, which allows you to implement Zero Trust for all of your infrastructure.
+tocDepth: 3
+---
+
+The Teleport Access Platform is a suite of software and managed services that
+delivers on-demand, least-privileged access to infrastructure on a foundation of
+cryptographic identity and Zero Trust, with built-in identity security and
+policy governance.
+
+## Use cases
+
+Organizations use the Teleport Access Platform to:
+
+- **Eliminate infrastructure and access silos:** Teleport provides a single
+  system for role-based access controls, audit, and access for all of your
+  infrastructure, from cloud provider APIs to Kubernetes clusters.
+- **Introduce Zero Trust with theft-resistant credentials:** Teleport
+  authenticates access to all of your infrastructure with short-lived
+  certificates, verified at every endpoint. You can roll out a Zero Trust
+  strategy for your organization by setting up Teleport alone.
+- **Address complex compliance needs:** Teleport allows you to satisfy
+  compliance frameworks like SOC 2 and FedRAMP with no need for additional
+  tooling or process changes. 
+
+## Products
+
+The Teleport Access Platform consists of three products:
+
+- **Teleport Access** provides on-demand, least privileged access, on a
+  foundation of cryptographic identity and Zero Trust.
+- **Teleport Identity** hardens your infrastructure with identity governance and
+  security.
+- **Teleport Policy** unifies and controls access policies across all of your
+  infrastructure.  
+
+### Teleport Access
+
+**Teleport Access** provides Zero Trust connectivity to all of your
+infrastructure. You can enable users to access servers, databases, and other
+infrastructure components over the public internet, even if those components are
+protected behind a firewall. 
+
+All infrastructure resources belong to a unified inventory, with a single
+role-based access controls system to allow for least-privilege access. You can
+enable users to authenticate to resources using Single Sign-On providers like
+GitHub and Google Workspace. Connectivity takes place through short-lived
```

Comment:
It'd be good to clarify that Teleport can act as the SSO or you can integrate your existing SSO. In terms of SSOs to mention, I think it's important to include Okta as that's what a lot of folks will be specifically looking for.